common-kafka/src/main/java/com/cerner/common/kafka/consumer/ProcessingPartition.java

```diff
@@ -79,7 +79,7 @@
     /**
      * The consumer for this partition
      */
-    private final Consumer<K, V> consumer;
+    private final KafkaConsumer<K, V> consumer;
```

Comment:
Why can we no longer use the `Consumer` interface? Would be very helpful for you to add some summary notes to the PR description about the necessary changes.

Reply:
I initially switched to using KafkaConsumer instead of the interface because I thought it was needed to get around the committed(TopicPartiton) deprecation.  Upon going back to find the reasoning I saw that the committed(Collection<TopicPartition>) method was available on the interface itself as well.  So, I have switched back to using the interface where it's possible to do so in: https://github.com/cerner/common-kafka/pull/51/commits/2c3e1f609b8dde350039702791b313fd77e1a972